tests/baselines/reference/outModuleConcatCommonjs.js.map

```diff
@@ -1,2 +1,2 @@
 //// [all.js.map]
-{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":";;;;;AACA,0BAA0B"}
\ No newline at end of file
+{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":";;;;;AACA,0BAA0B;;AAE1B;;;;KAAkB;AAAlB,YAAa,CAAC,CAAI;;ACHlB,6BAA0B;AAC1B;;;;;;EAAuB,KAAC,GAAI;AAA5B,YAAa,CAAC,CAAc"}
```

Comment:
This is another case where I'd rather we not accept this file as it uses `outFile`.